### DISCUSSION TITLE
Hat die Anzahl an Schreibzyklen einen Einfluß auf die Lebensdauer der Wechselrichter?

### DISCUSSION THREAD

**Discussion by ELeichstetten** [tbnobody/OpenDTU#2011] [Q&A] [answered]
Update: Nachdem ich die Frage gestellt habe, habe ich die Antwort im Forum gefunden. (passende "Reizworte")
Unter "MQTT limit_nonpersistent_absolute #431" wurde das Thema bereits behandelt.
Ich lasse die Frage trotzdem mal stehen, jedoch kann sie auch gerne gelöscht werden.

Erstmal ein "Hallo" an alle in diesem Forum.

Mich würde interessieren, ob die Anzahl an Schreibzyklen (durch Dynamic Power Limiter) einen Einfluß auf die Lebensdauer der Wechselrichter hat.
Ein Limit, das dauerhaft gesetzt wird, wird sicherlich in einem gegen Spannungsausfall gesicherten Bereich ("ROM") gespeichert.
Ein temporäres Limit kann entweder im "RAM" oder aber auch im "ROM" abgelegt sein.
In letzterem Fall kann es zu einem Verschleiß der Speicherzellen durch permanentes Umschreiben kommen.
Deshalb die Frage vom Eingang: Werden die Limits des Dynamic Power Limiters im "RAM" oder im "ROM" des Wechselrichters hinterlegt?

Ich hoffe, ich habe das leidlich verständlich ausgedrückt.
Vielleicht kennt da jemand die Antwort drauf.

**Comment by MetaChuh** [comment, marked as answer]
@ELeichstetten 

> Deshalb die Frage vom Eingang: Werden die Limits des Dynamic Power Limiters im "RAM" oder im "ROM" des Wechselrichters hinterlegt?

ram.
dpl werte werden nicht im inverter flash gespeichert.

**Reply by ELeichstetten** [reply]
Vielen Dank für die prompte Rückmeldung.

**Comment by jubie25** [comment]
Das muss ich widerlegen.

Hier:
Habe einen HM-800-2T über OpenDTU angeschlossen. Firmware der Hoymiles ist 1.0.10 mit Datumscode 2021-11-01 08:42:00. Und ich kann bestätigen, dass **nicht-persistent gesetzte Werte auch in den persist-Speicher gespeichert werden**.
Test: 
1. Home Assistant, setze beide Werte auf 800W und setze danach den nicht persistenten Wert auf 47W. Beobachte die Stromerzeugung in OpenDTU: passt an die vorübergehend festgelegte Grenze. Limit wird auch in OpenDTU mit 47 W angezeigt.
2. Hoymiles vom Strom trennen (AC und beide Platten). Warte auf Stop des LED-Blinking und warte zusätzliche 30 Sekunden. 
3. Verbinden alles wieder
4. OpenDTU erhält seine ersten Werte wieder von Hoymiles, die 47W-Grenze wird angezeigt. Produzierte Leistung passt ebenfalls dazu.
5. Home Assistant zeigt wieder 47W als Limit, für persistentes und nicht persistentes Limit.

Test 2: Zurücksetzen von Hoymiles per Neustart-Befehl in OpenDTU: Dadurch werden beide Werte auf Standard (800W) zurückgesetzt. **Das ist NICHT dasselbe wie einfache Trennung von allen Stromquellen**!

Es ist also klar, dass diese nicht persistent gesetzten Grenzwerte trotzdem persistent gespeichert werden!

Eine Automation, die das non-persist-Limit periodisch anpasst, wird das Flash des Wechselrichters also binnen Wochen zerstören.

Hilft ein Firmware-Update? Hat jemand Erfahrung, die weiterhilft?

**Comment by jubie25** [comment]
O M G!!!
Ich habe die falsche Hoymiles-Einheit getestet und ge-power-cycled. Mit der richtigen kann ich bestätigen, dass die Werte flüchtig sind und vermutlich nicht das Flash berühren. 
Sorry!!
